versions/build.gradle.kts

```diff
@@ -32,6 +32,10 @@ dependencies {
     api("org.junit.jupiter:junit-jupiter")
     api("org.junit.platform:junit-platform-launcher")
 
+    api(platform("tools.jackson:jackson-bom:3.0.2"))
```

Comment:
The version has to go to below to the constraints (doesn't it?)

Reply:
I have absolutely no idea, because it's a bom and other boms are declared at the top as well